src/Tasks.UnitTests/Exec_Tests.cs

```diff
@@ -141,13 +141,13 @@ public void TimeoutFailsEvenWhenExitCodeIsIgnored()
                 // The standard check for SIGTERM fails intermittently on macOS Mono
                 // https://github.com/dotnet/msbuild/issues/5506
                 // To avoid test flakiness, allow 259 even though I can't justify it.
-                exec.ExitCode.ShouldBeOneOf(143, 259);
+                exec.ExitCode.ShouldBeOneOf(137, 259);
```

Comment:
Could you please see in https://github.com/dotnet/msbuild/issues/5506 if you have any insights about that?

Reply:
I've removed the timeout on `WaitForExit` which caused this test to fail (https://github.com/dotnet/msbuild/issues/5506#issuecomment-770654527) and updated the test accordingly.